froncik/.env.local

```diff
@@ -1 +1 @@
-# NEXT_PUBLIC_API_PATH=http://127.0.0.1:1234/majooo #default /api
+NEXT_PUBLIC_API_PATH=http://127.0.0.1:8080/api #default /api
```

Comment:
As the DEV.md states for running:
![Wed 30 Nov 15:57:18 CET 2022](https://user-images.githubusercontent.com/10052198/204831816-4fdaf18f-af10-4b39-a65b-b5a8a188bb58.png)

Changing this line will probably break it for the production application.
I may fix it today to modify this variable for production.

For now please remove.

Reply:
DONE